operator/metrics/metrics.go

```diff
@@ -50,7 +50,7 @@ func Register() {
 		Handler: m,
 	}
 
-	shutdownCh = make(chan bool)
+	shutdownCh = make(chan struct{})
```

Comment:
Ah I see that it was changed here. Please squash this change into the original commit. There's no need to have diffs between commits in a PR.